src/main/java/com/amihaiemil/versioneye/RtVersionEye.java

```diff
@@ -82,5 +85,12 @@ public Services services() {
     public Users users() {
         return new RtUsers(this.entry);
     }
-
+    
+    @Override
+    public VersionEye trusted() throws IOException {
+        return new RtVersionEye(
+            this.entry.method(Request.GET).through(TrustedWire.class)
```

Comment:
@SherifWaly Is it mandatory to set the ``GET`` method before calling ``through(...)``? It is by default ``GET`` so it makes no sense to set it.